src/dft/src/clock_domain/ClockDomain.cpp

```diff
@@ -63,10 +65,22 @@ std::string_view ClockDomain::getClockEdgeName() const
   return "Unknown clock edge";
 }
 
+namespace {
+size_t FNV1a(const uint8_t* data, size_t length)
+{
+  size_t hash = 0xcbf29ce484222325;
+  for (size_t i = 0; i < length; i += 1) {
+    hash *= 0x00000100000001b3;
+    hash ^= data[i];
+  }
+  return hash;
+}
+};  // namespace
+
 size_t ClockDomain::getClockDomainId() const
 {
-  return std::hash<std::string_view>{}(clock_name_)
-         ^ std::hash<ClockEdge>{}(clock_edge_);
+  return FNV1a((uint8_t*) clock_name_.data(), clock_name_.length())
+         ^ FNV1a((uint8_t*) &clock_edge_, sizeof(ClockEdge));
```

Comment:
Use boost::hash_combine

Reply:
I used hash_mix because hash_combine assumes that you're using boost::hash.